docs/application-development/features.md

```diff
@@ -344,7 +344,7 @@ Content is not available
 ```ts
 import {setHandler} from "@temporalio/workflow";
 
-export async function myWorkflow() {
+export async function YourWorkflow() {
```

Comment:
Lowercase

Reply:
@rachfop I am not considering function names to be placeholders and so I don't think that style guidance applies here. Though there could be other style guidance I am not following - so we could check up on that.

@lorensr This was my call - I want our app dev guide code samples to be explicit about what is 100% your code - your functions names - your custom data structures - etc.
My thinking on is that it helps to reduce reader fatigue - and clearly illustrate which key words are part of the SDK within the context of the things that the user is creating.

This is a content design choice I implemented in the legacy docs for the Go SDK - which as a whole have been receiving great feedback in our surveys. I can't make 100% correlation there - but my instinct is that is a good habit for us to stick with.